pages/docs/concepts/asyncapi-document/extending-specification.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Extending Specification
+weight: 240
+---
+
+Extending the AsyncAPI specification is a technique that allows developers to include domain-specific or use-case-specific information not supported by the base specification. This extension capability provides customization, enabling APIs to accommodate unique details that would not otherwise fit within the confines of the standard AsyncAPI specification.
+
+The benefits of extending the AsyncAPI specification include enhanced specificity — delivering a clear encoding of necessary API details to applications, and consistency — allowing reusable components across the API. However, extensions should be used sparingly. Since they go beyond the Specification, they may not be universally understood.
+
+## Specification Extensions
```

Comment:
We observe sentence capitalization for headers in our docs, please update all your headers accordingly 😄